js/pages/tola_management_pages/user/components/edit_user_programs.js

```diff
@@ -597,7 +581,7 @@ export default class EditUserPrograms extends React.Component {
                                     id: rowData.id,
                                     disabled: is_role_disabled(rowData),
                                     type: rowData.type,
-                                    options: rowData.options,
+                                    options: this.getRoleOptions.call(this, rowData),
```

Comment:
It would have taken me forever to figure out that I needed to use a `call` method here.  I still don't grok JS on some fundamental level.

Reply:
I definitely got to there by my usual path of not using .call, fighting with the errors, finally thinking to console.log(this), facepalm, add .call.  I use .call and .bind as complete hail marys

Reply:
That's my SOP as well.